### PR TITLE
create trip geometries table and trigger

### PR DESCRIPTION
**Pull request by hunterowens**
don't merge until I do the same for status_changes, but should show what is on the db now

### CODE REVIEW

**Comment by hunterowens** [comment]
this PR is ready for review @ian-r-rose

File: dags/transportation/dockless/dockless_elt.py [hunk 1]
```diff
@@ -217,6 +217,7 @@ def load_to_s3(**kwargs):
 status_changes = """
 
 CREATE TABLE IF NOT EXISTS status_changes (
+    id SERIAL PRIMARY KEY
```
Comment (ian-r-rose) [inline dags/transportation/dockless/dockless_elt.py]: This is unfortunate. I know you said there was some history here -- any forthcoming ways to more reliably assign a status change id?

File: dags/transportation/dockless/geom_parse.sql [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+	points GEOMETRY 
+); 
+/* process route function */ 
+CREATE OR REPLACE FUNCTION process_route(jsonb) 
+	RETURNS geometry AS 
+	$func$
+	BEGIN
+		RETURN ST_MakeLine(ARRAY(
+					SELECT ST_GeomFromGeoJSON((feature -> 'geometry')::text) as geometry FROM (
+						SELECT jsonb_array_elements($1 -> 'features') as feature
```
Comment (ian-r-rose) [inline dags/transportation/dockless/geom_parse.sql]: Holy indentation batman. Does your SQL formatter really like tabs?

File: dags/transportation/dockless/geom_parse.sql [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+	END
+	$func$  
+    LANGUAGE plpgsql IMMUTABLE;
+
+/* trigger function */ 
+CREATE OR REPLACE FUNCTION status_geom_copy() RETURNS TRIGGER AS
+$BODY$
+BEGIN
+    INSERT INTO
+        status_change_geoms(status_change_id, event_location_geom)
```
Comment (ian-r-rose) [inline dags/transportation/dockless/geom_parse.sql]: As long as we're here, should we add any of the other fields to the `trips_geom` and `status_change_geom` tables? Is that data going to break the bank?

Comment (hunterowens) [reply dags/transportation/dockless/geom_parse.sql]: status_change_geoms and trips_geoms get joined to the original tables in the materialized view update (see #72).

File: dags/transportation/dockless/geom_parse.sql (hunk 2 above)
Comment (hunterowens) [reply dags/transportation/dockless/geom_parse.sql]: hahahaha I'll update vscode.

File: dags/transportation/dockless/dockless_elt.py (hunk 1 above)
Comment (hunterowens) [reply dags/transportation/dockless/dockless_elt.py]: yeah, unfortunately there isn't a status_change_id in the spec. 

the alternate approach is joining on the constraints from the CREATE_TABLE statements, but those haven't been entirely reproduceable

**Comment by hunterowens** [comment]
@ian-r-rose my intent is to rerun the script as required, but it would be possible to schedule them via airflow if that is valuable, they should be fairly ideomatic.

File: dags/transportation/dockless/geom_parse.sql (hunk 3 above)
Comment (ian-r-rose) [reply dags/transportation/dockless/geom_parse.sql]: Ah, makes sense.

**Comment by ian-r-rose** [comment]
I don't see much reason to schedule it right now -- updating as necessary makes sense to me since this probably won't be changing too much (he said naively).

**Pull request merged**